src/commands/register.ts

```diff
@@ -360,4 +364,37 @@ export default class ComponentRegister extends BaseCommand {
   public static getTagFromFlags(flags: any): string {
     return flags.environment ? `${ENV_TAG_PREFIX}${flags.environment}` : flags.tag;
   }
+
+  private async generateDependenciesWarnings(component_dependencies: Dictionary<string>, account_id: string) {
+    const existing_account_components: Dictionary<string> = {};
+    const { rows, _ } = (await this.app.api.get(`/accounts/${account_id}/components`)).data;
+    for (const row of rows) {
+      existing_account_components[row.component.name] = row.tag;
+    }
+
+    const register_warnings: string[] = [];
+    const latest_tag_warnings: string[] = [];
+    for (const key of Object.keys(component_dependencies)) {
+      if (!existing_account_components[key]) {
+        register_warnings.push(key); 
+      } 
+      if (component_dependencies[key] !== 'latest') {
+        latest_tag_warnings.push(key);
+      }
+    }
+
+    const warning = chalk.hex('#FFA500');
```

Comment:
Why use this instead of `chalk.yellow()`?

Reply:
It is a warning, so I thought I should use yellow. Should I use a different color instead?

Reply:
I agree that it should be yellow, but the shade of yellow should be consistent throughout the rest of the CLI. For example, most if not all places, this is the yellow that's used

https://github.com/architect-team/architect-cli/blob/rc/src/commands/scale.ts#L144

Reply:
Ahh ok. I'll use that instead.